kind/README.md

```diff
@@ -0,0 +1,41 @@
+# kind Environment
+
+## Overview
+
+kind is a tool for running local Kubernetes cluster with nodes deployed as Docker containers. It can be deployed as a multi-node container what gives as ability to test in a environment close to production cluster.
+
+## Directory structure
+
+The `kind` folder contains a set of configuration files for the kind cluster.
+
+Its structure looks as follows:
+
+```
+  ├── cluster                  # kind cluster configuration
+        └── resources          # Default resources applied during cluster setup
+  ├── config                   # Kyma overrides for kind
+  ├── scripts                  # Scripts used for kind environment
+  └── KIND_KUBERNETES_VERSION  # File with Kubernetes version used for cluster
+```
+
+## Deploy Kyma cluster on kind
+
+To deploy Kyma on kind, execute:
+
+```bash
+./scripts/install-kyma.sh
+```
+
+To list available options, execute:
+
+```bash
+./scripts/install-kyma.sh --help
+```
+
+It may require more resources for Docker, adjust it in Docker preferences (cpu and memory)
```

Comment:
I suggest to be more precise here, what values are recommended to set in docker preferences

Reply:
```

> **TIP:** You may have to adjust CPU and memory settings for Docker. Use the recommended values of 6 CPU and 12 GB. Adjust these values in Docker preferences. 

```